vendor/synopsys/Makefile

```diff
@@ -0,0 +1,65 @@
+LIBRARIES = ../../../libraries
+
+LIB_NAME = sky130_fd_sc_hd
+
+ROOT = $(realpath .)
+SCRIPTS = $(ROOT)/scripts
+RESULTS_DIR = $(ROOT)/results
+LIBS_DIR = $(RESULTS_DIR)/lib
+TECH_DIR = $(RESULTS_DIR)/tech
+OUTPUT_DIR = $(LIBS_DIR)/$(LIB_NAME)
+LOGS = $(ROOT)/logs
+
+COMMON_OPTIONS = 'set target_lib_dir $(LIBRARIES)/$(LIB_NAME)/latest; set target_lib_name $(LIB_NAME); set output_dir $(OUTPUT_DIR)'
+LC_OPTIONS = -x $(COMMON_OPTIONS)
+LM_OPTIONS = -x $(COMMON_OPTIONS)
+MW_OPTIONS = $(COMMON_OPTIONS)
+
+all: $(LIB_NAME)_ndm tech
+
+setup:
+	test -d work || mkdir work
+	test -d $(OUTPUT_DIR) || mkdir -p $(OUTPUT_DIR)
+	test -d $(LOGS) || mkdir $(LOGS)
+	test -d $(TECH_DIR) || mkdir -p $(TECH_DIR)
+	date > $@
+
+$(LIB_NAME)_db: setup
+	cd work && lc_shell -f $(SCRIPTS)/lc.tcl $(LC_OPTIONS) |& tee $(LOGS)/lc_$@.log
```

Comment:
The |& before tee result in an error:
/bin/sh: 1: Syntax error: "&" unexpected

It works after removing the &
`cd work && lc_shell -f $(SCRIPTS)/lc.tcl $(LC_OPTIONS) |& tee $(LOGS)/lc_$@.log`